src/partials/30_section-features.html

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <div class="row">
       <div class="col">
-        <div class="feature-box active">
+        <div class="feature-box" href="#">
           <div class="icon"><i class="fas fa-truck"></i></div>
```

Comment:
Dodanie atrybutu `href` nie czyni elementu linkiem. To jest wciąż `div`, a powinien by `a`.